stack.yaml

```diff
@@ -14,22 +14,25 @@ extra-deps:
     https://gitlab.com/morley-framework/morley.git
     # ^ CI cannot use ssh, so we use http clone here
   commit:
-    9e69d52741e456bd0976a1ec4c45cc4c43a11722 # morley-1.11.1
+    # 56d74ababeda9aba8e194783fc5e395fa7a6b69a # morley-1.12.0
+    4614429f28a37eff5cd442a94213b117c7140fb3 # heitor.toledo/#440-allow-funds-from-contracts-with-zero-amount
```

Comment:
It seems that this is close to being solved, let's wait for it, hopefully it won't take much time.

Reply:
Let's hope that no breaking changes will be merged before it :joy:

Reply:
Ok, now that that MR has been merged I think we should merge this as well.

There's probably going to be an updated `morley` soon, but I'd rather not wait, we can always modify that later.